src/react-guides/Guides.tsx

```diff
@@ -523,4 +519,14 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
             selectedGuides:[],
         });
     }
+
+    private calcHorizontalTransform(offsetX: number, nextPos: number) {
+        const displayPos = [offsetX, nextPos];
+        return `translate(${displayPos.map((v, i) => !i ? '-6px' : `${v - 10}px` ).join(', ')}) rotate(-90deg)`;
```

Comment:
change map function arguments to the full name, which describes what the argument for

Reply:
having function arguments names `offsetX`, `offsetY` will describe what they do